### PR TITLE
Flustudy us v2 setup

### PR DESCRIPTION
**Pull request by goldjay**
[FEV-1111](https://auderenow.atlassian.net/browse/FEV-1111)

* Moved flustudy_us to archive folder
* Copied flustudy_au and made some naming changes

I ran through iOS and Android flows and it looks like it's all working.

EDIT: Try #2 

* Started fresh from master, git mv FluStudy_us to archive/FluStudy_us, revcycle FluStudy_au to FluStudy_us
* Applied all of @tpaik18 requested changes

### CODE REVIEW

**Comment by tpaik18** [comment]
@rightparen can correct me if I'm wrong but @goldjay I believe we don't want to check in google-services.json or GoogleService-Info.plist as these contain our API keys. These should go into LastPass and engineers can install them locally and I can make sure CircleCI can inject them into the release build.

**Comment by goldjay** [comment]
@tpaik18 Thought they were covered by the .gitignore. I'll remove them.

**Comment by rightparen** [comment]
@goldjay thanks for working through this.

Could you give the steps you followed to generate this?  Was it something like this?  https://github.com/AudereNow/learn/pull/1040#issuecomment-490302938

File: FluStudy_us/android/README.md [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
-    so. You might also be asked to update Google Play Services, which you should.
-
-Note: at the moment, the Android simulator isn't yet working with localapi... so you must
-`yarn start` instead of `yarn start-localapi` if you want to run in the simulator.
+1.
+Download and install Android Studio
+
+2.
+Install Android SDK 27 (Oreo) by opening Tools -> SDK Manager or from the starting window choose Configure -> SDK Manager at the bottom.
+Check the SDK 27 (Oreo) box and OK/accept the dialog.
```
Comment (tpaik18) [inline FluStudy_us/android/README.md]: The compile SDK is now 28 (was 27 for Fever) so the instructions should be to install Android SDK 28.

File: FluStudy_us/ios/fastlane/Appfile [hunk 2]
```diff
@@ -3,9 +3,9 @@ itc_team_id("119446930") # App Store Connect Team ID
 team_id("G8UCCFY788") # Developer Portal Team ID
 
 for_lane :debug do
-    app_identifier "org.auderenow.fluathome.dev"
+    app_identifier "org.auderenow.fluathome-au.dev"
 end
```
Comment (tpaik18) [inline FluStudy_us/ios/fastlane/Appfile]: app_identifier in this file and in ios/fastlane/Fastfile needs to match the bundle ID used earlier which is org.auderenow.fluathome-us and not org.auderenow.fluathome-au.

File: FluStudy_us/ios/fastlane/Fastfile [hunk 3]
```diff
@@ -143,22 +144,25 @@ platform :ios do
...
     android_build_file = "FluStudy_us/android/app/version.properties"
-    Helper.backticks("sed -i '' \"s/\\(VERSION_CODE=\\).*\\$/\\1#{build_num}/\" ../../../#{android_build_file}")
+    Helper.backticks("rm ../../../#{android_build_file}")
+    Helper.backticks("echo VERSION_CODE=#{build_num} > ../../../#{android_build_file}")
     if versionBump 
       version = increment_version_number
       Helper.backticks("yarn version --patch --no-git-tag-version")
-      message = "Build script incrementing version to #{version}, build number to #{build_num}"
-      include = ["FluStudy_us/package.json", "#{android_build_file}"]
+      message = "Build script incrementing Cough version to #{version}, build number to #{build_num}"
```
Comment (tpaik18) [inline FluStudy_us/ios/fastlane/Fastfile]: Cough -> Chills

File: FluStudy_us/ios/fastlane/Fastfile [hunk 4]
```diff
@@ -143,22 +144,25 @@ platform :ios do
...
+      include = ["flustudy_us/package.json", "#{android_build_file}"]
     else 
-      message = "Build script incrementing build number to #{build_num}"
+      message = "Build script incrementing Cough build number to #{build_num}"
```
Comment (tpaik18) [inline FluStudy_us/ios/fastlane/Fastfile]: Cough -> Chills

File: FluStudy_us/ios/fastlane/Matchfile [hunk 5]
```diff
@@ -3,7 +3,7 @@ git_url "git@github.com:AudereNow/signing"
 
 type "development" # The default type, can be: appstore, adhoc, enterprise or development
 
-app_identifier ["org.auderenow.fluathome.dev", "org.auderenow.fluathome"]
+app_identifier ["org.auderenow.fluathome-au.dev", "org.auderenow.fluathome-au"]
```
Comment (tpaik18) [inline FluStudy_us/ios/fastlane/Matchfile]: org.auderenow.fluathome-us

**Comment by goldjay** [comment]
@rightparen Yes, I did not do the git rev-parse step though.
mkdir archive
git mv flustudy_us/* archive
mkdir flustudy_us
git mv flustudy_au/* flustudy_us

**Comment by rightparen** [comment]
```
mkdir archive
git mv flustudy_us/* archive
mkdir flustudy_us
git mv flustudy_au/* flustudy_us
```
@goldjay this doesn't look like the list of commands you ran.  These look incorrect, but the current contents of the branch don't reflect the errors in the commands here.  (E.g. it looks like you did `git mv FluStudy_us archive` instead of `git mv flustudy_us/* archive`.  E.g. `mkdir flustudy_us` would create the directory in lower-case.  E.g. `git mv flustudy_au/* flustudy_us` would have deleted FluStudy_au contents.)

Were the steps you gave a hand-written approximation?  I was hoping you could post the output of `history` that was relevant so I could actually review the command-line steps you did.

**Comment by goldjay** [comment]
@rightparen They were an approximation. I'll redo the transfer using the steps you linked.

**Pull request merged**